src/Tests/Microsoft.NET.Perf.Tests/BuildPerf.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Microsoft.Xunit.Performance.Api;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Perf.Tests
+{
+    public class BuildPerf : SdkTest
+    {
+        public BuildPerf(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        //  These tests are currently disabled for full framework MSBuild because the CI machines don't
+        //  have an MSBuild that supports the /restore command-line argument
+        //  Also, Microsoft.Xunit.Performance.Api.ProcessExitInfo doesn't handle non-Windows
+        //  information gathering - disabling for non-Windows
```

Comment:
File a bug in https://github.com/Microsoft/xunit-performance for this, and link to it from this comment.